static/app/components/events/interfaces/spans/scrollbarManager.tsx

```diff
@@ -233,11 +237,70 @@ export class Provider extends Component<Props, State> {
   hasInteractiveLayer = (): boolean => !!this.props.interactiveLayerRef.current;
   initialMouseClickX: number | undefined = undefined;
 
-  onScroll = () => {
+  onWheel = (deltaX: number) => {
     if (this.isDragging || !this.hasInteractiveLayer()) {
       return;
     }
 
+    // Setting this here is necessary, since updating the virtual scrollbar position will also trigger the onScroll function
+    this.isWheeling = true;
+
+    if (this.wheelTimeout) {
+      clearTimeout(this.wheelTimeout);
+    }
+
+    this.wheelTimeout = setTimeout(() => {
+      this.isWheeling = false;
+      this.wheelTimeout = null;
+    }, 200);
+
+    const interactiveLayerRefDOM = this.props.interactiveLayerRef.current!;
```

Comment:
`!`

Reply:
I think it's safe to assume it's not null/undefined here, since the wheel event shouldn't be able to occur otherwise